### PR TITLE
feat: add ValueItem component

### PR DESCRIPTION
**Pull request by klubiniecki-qantas**
## Description

Adds ValueItem component.

## Looks like this

Screen Shot 2022-02-07 at 12 23 05 pm

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [x] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), [Marcel](https://github.com/Anthropic), [Sam](https://github.com/SamuSan) [Long](https://github.com/KieraDOG)) or a Roo UI working-group human.
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

File: src/components/ValueItem/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+## Example
+
+```js
+import { ValueItem } from 'roo-ui/components';
+
+export default (
+  <ValueItem
+    image={{
```
Comment (eneo5541) [inline src/components/ValueItem/README.md:18]: Example indicates the prop is called `image` but code is looking for a prop named `icon`

File: src/components/ValueItem/ValueItem.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+ValueItem.propTypes = {
+  icon: PropTypes.shape({
+    src: PropTypes.string.isRequired,
+    alt: PropTypes.string.isRequired,
+  }).isRequired,
+  heading: PropTypes.string.isRequired,
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+  layout: PropTypes.oneOf(['row', 'column']),
+  maxWidth: PropTypes.string,
```
Comment (eneo5541) [inline src/components/ValueItem/ValueItem.js:61]: `maxWidth` should accept string, number or array proptypes.

File: src/components/ValueItem/ValueItem.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, Flex, Image, Heading } from 'components';
+import { rem } from 'polished';
+import styled from '@emotion/styled';
+import { color } from 'styled-system';
+
+const ContentWithLink = styled(Text)`
+  a {
```
Comment (eneo5541) [inline src/components/ValueItem/ValueItem.js:9]: Sorry, what is the purpose of this? Shouldn't the color of anchor tags be determined by the `children` components that are passed in?

File: src/components/ValueItem/README.md (hunk 1 above)
Comment (klubiniecki-qantas) [reply src/components/ValueItem/README.md:18]: 👍

File: src/components/ValueItem/ValueItem.js (hunk 2 above)
Comment (klubiniecki-qantas) [reply src/components/ValueItem/ValueItem.js:61]: 👍

File: src/components/ValueItem/ValueItem.js (hunk 3 above)
Comment (klubiniecki-qantas) [reply src/components/ValueItem/ValueItem.js:9]: When talking to designers, they pointed that the most common `children` here will be either text or text with inline link. This is to cover for this use case, but that could be left out and handled by implementation. I can go with your suggestion here 👍

**Review by eneo5541 — APPROVED** [review]

**Pull request merged**